defaults.go

```diff
@@ -0,0 +1,7 @@
+package storageredis
+
+const (
+	DefaultRedisHost = "127.0.0.1"
+	DefaultRedisPort = "6379"
+	DefaultKeyPrefix = "caddy"
+)
```

Comment:
These three constants are already defined (using slightly more generic names) here: https://github.com/pberkel/caddy-storage-redis/blob/324d7103c21d9f5e97b1a1c3bd101752eaef7839/storage.go#L24. Could I ask you to update your code changes to reference the existing constants instead rather than introducing new ones?

Reply:
ok, should be done